py/selenium/common/exceptions.py

```diff
@@ -310,6 +310,6 @@ class SessionNotCreatedException(WebDriverException):
 
 class UnknownMethodException(WebDriverException):
     """
-    The requested command matched a known URL but did not match an method for that URL.
+    The requested command matched a known URL but did not match a method for that URL.
```

Comment:
Actually, shouldn't it be `any`
```suggestion
    The requested command matched a known URL but did not match any method for that URL.
```